src/main/cpp/sensors/ValorCurrentSensor.cpp

```diff
@@ -17,6 +17,17 @@ void ValorCurrentSensor::setSpikeSetpoint(double _setpoint)
     spikedSetpoint = _setpoint;
 }
 
+void ValorCurrentSensor::setCacheSize(int _size)
+{
+    if (_size < cacheSize) {

```

Comment:
This code already exists in `reset()`. As long as we call:
```
cacheSize = _size;
reset();
```
We should be covered